msbuild/Xamarin.Shared/Xamarin.Shared.props

```diff
@@ -210,6 +210,9 @@ Copyright (C) 2020 Microsoft. All rights reserved.
 		<EnableOnDemandResources Condition="'$(_PlatformName)' == 'macOS' And '$(EnableOnDemandResources)' == ''">false</EnableOnDemandResources>
 		<EnableOnDemandResources Condition="'$(_PlatformName)' != 'macOS' And '$(EnableOnDemandResources)' == ''">true</EnableOnDemandResources>
 
+		<!-- Enable/Disable default ruuntime features -->
```

Comment:
typo `ruuntime`
not sure the comment is really needed

Reply:
My keyboard keeps sticking on me!